runtime/acala/src/xcm_config.rs

```diff
@@ -133,17 +135,20 @@ parameter_types! {
 		).into(),
 		aca_per_second()
 	);
-	pub ForeignAssetUnitsPerSecond: u128 = aca_per_second();
-	pub AcaPerSecondAsBased: u128 = aca_per_second();
+	pub NativeTokenPerSecond: u128 = aca_per_second();
```

Comment:
```suggestion
	pub BaseRate: u128 = aca_per_second();
```

Same to other runtime.